modules/auth/proc-adding-google-as-an-authentication-provider.adoc

```diff
@@ -0,0 +1,59 @@
+[id='proc-adding-google-as-an-authentication-provider_{context}']
+= Enabling the Google authentication provider
+
+{product} includes a Google authentication provider that can authenticate users by using Google OAuth2.
+
+.Procedure
+. To allow {product-short} to authenticate with Google, create OAuth credentials in Google.
+For more information, see the official Google Cloud documentation for link:https://developers.google.com/identity/openid-connect/openid-connect#appsetup[Setting up OAuth 2.0].
++
+.. To authorize {product-short} to access the user consent screen, enter the necessary information on the https://console.cloud.google.com/apis/credentials/consent/edit[*Google Cloud > Oauth consent screen > Edit app registration*] page, including the following:
++
+... On the *OAuth consent screen* tab, enter your {product-short} domain (_<APP_FQDN>_) into the *Authorized domains* field.
+... On the *Scopes* tab, select the following scopes:
++
+openid:: Associate you with your personal info on Google.
+auth/userinfo.email:: See your primary Google Account email address.
+auth/userinfo.profile:: See your personal info, including any personal info you've made publicly available.
+... On the *Test users* tab, add yourself as a test user, if using the External user type.
```

Comment:
```suggestion
* *openid* associates you with your personal info on Google.
* *auth/userinfo.email* shows your primary Google Account email address.
* *auth/userinfo.profile* shows your personal info, including any personal info you've made publicly available.
... On the *Test users* tab, add yourself as a test user, if using the External user type.
```

Reply:
This follows the approved "Simple list" format for definition lists per the CCS style guide